string goal_topology
---
uint16 error_code
string error_msg
---
GridCell[] current_poses
builtin_interfaces/Duration navigation_time
int32 num_goals_reached
